templates/thanos-values.yaml.tpl

```diff
@@ -1,13 +1,21 @@
-
-existingServiceAccount: prometheus-operator-kube-p-prometheus
-
 metrics:
   enabled: true
   serviceMonitor:
     enabled: true
 
 storegateway:
+  resources:
+    limits:
+      cpu: 1600m
+      memory: 24Gi
+    requests:
+      cpu: 10m
+      memory: 100Mi
+
   enabled: true
+  serviceAccount:
```

Comment:
because in v13:

```
This version also removes deprecated service port values like receive.service.http.port in favor of recieve.service.ports.http, as well as existingServiceAccount.
```